src/openbf.f

```diff
@@ -138,14 +138,14 @@ RECURSIVE SUBROUTINE OPENBF(LUNIT,IO,LUNDX)
       CHARACTER*(*) IO
       CHARACTER*255 FILENAME,FILEACC
       CHARACTER*128 BORT_STR,ERRSTR
-      CHARACTER*30  CPRINT(0:4)
+      CHARACTER*28  CPRINT(0:4)
 
       DATA          CPRINT/
      . ' (only ABORTs)               ',
      . ' (limited - default)         ',
      . ' (all warnings)              ',
-     . ' (all warning+informationa l)', 
-     . ' (low level high volume prnt)'/ 
+     . ' (all warning+informational)', 
+     . ' (low level high volume out)'/ 
```

Comment:
You still need to reduce the sizes of the first 3 members of CPRINT too.  They're all still 29 bytes.

For iprt =2, how about `' (all warnings+infos  )'` and for iprt =3 `' (all warnings+infos+debugs)'`?  That's more consistent and still keeps everything at or below 28 characters.

BTW, please also update the openbf() docblock as well to reflect this new iprt=3 option, under the documentation for argument LUNDX when IO='QUIET'